engine/src/main/java/org/hibernate/validator/constraints/SafeHtml.java

```diff
@@ -65,6 +65,12 @@
 	Tag[] additionalTagsWithAttributes() default { };
 
 	/**
+	 * @return Base URI to be used in resolving relative links to absolute ones.
```

Comment:
I suggest something like:
`Base URI used to resolve relative URIs to absolute ones. If not set, validation of HTML containing relative URIs will fail.`